src/version_definitions/java_1_13/entities/entity.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": {
+    "tag_type": "string",
+    "default": "minecraft:unknown_entity"
+  },
+  "Pos": {
+    "tag_type": "list",
+    "schema": {
+      "tag_type": "double"
+    }
+  },
+  "Motion": {
+    "tag_type": "list",
+    "schema": {
+      "tag_type": "double"
+    }
+  },
+  "Rotation": {
+    "tag_type": "list",
+    "schema": {
+      "tag_type": "float"
+    }
+  },
+  "FallDistance": {
+    "tag_type": "float"
+  },
+  "Fire": {
+    "tag_type": "short",
+    "default": -20
+  },
+  "Air": {
+    "tag_type": "short"
+  },
+  "OnGround": {
+    "tag_type": "byte"
+  },
+  "NoGravity": {
+    "tag_type": "byte"
+  },
+  "Dimension": {
+    "tag_type": "int"
+  },
+  "Invulnerable": {
+    "tag_type": "byte"
+  },
+  "PortalCooldown": {
+    "tag_type": "int"
+  },
+  "UUIDMost": {
+    "tag_type": "long"
+  },
+  "UUIDLeast": {
+    "tag_type": "long"
+  },
+  "CustomName": {
+    "tag_type": "string"
+  },
+  "CustomNameVisible": {
+    "tag_type": "byte"
+  },
+  "Silent": {
+    "tag_type": "byte"
+  },
+  "Passengers": {
+    "_tag_type": "list",
+    "schema": {
+      "tag_type": "compound",
+      "structure": "any"
```

Comment:
Why is the structure `any`? Seems like the structure is supposed to be `entity`.
Also seems like it's the only use-case for `structure: any`. If we can replace it with entity, maybe we can rid of the option to have `any` as a structure?

Reply:
That would sadly cause it to continually recursive try to nest entity templates and would eventually crash python. So I decided to create the `any` tag to resolve the problem and also allow flexibility in the future.

Reply:
Do you think there is a way to solve those types of cases or should we just leave it like that? I'll go with whatever you think is best here, I just think that if we'll make it `entity` rather than `any` it will allow us to enforce the structure of it a bit better. If you think it will be better to leave it as "any", you can just mark this discussion as resolved.

Reply:
If we were to change it back to `entity` it would be recursive. We could change how applying templates work and have something like `TemplateLoader` check the tag type and then load and apply the structure there, I already tried that once it got a little complex, but the code is more straightforward than it was then, so I'll test it